R/ids_with_token.R

```diff
@@ -27,7 +27,7 @@
 #' entry of the list of source expressions. Indices correspond to the
 #' *rows* where `fun` evaluates to `TRUE` for the `value` in the *token* column.
 #' @export
-ids_with_token <- function(source_expression, value, fun = `==`, source_file) {
+ids_with_token <- function(source_expression, value, fun = `==`, source_file) { # nolint: function_argument_linter.
```

Comment:
Instead, set the default to `source_file = NULL`

Reply:
Thanks, done.